src/Makefile

```diff
@@ -1500,90 +1507,21 @@ EXTRA_CFLAGS += -fPIC
 RTOBJS := $(sort $(foreach mod,$(obj-m),$(call TORTOBJS,$(mod))))
 
 RTDEPS := $(sort $(patsubst $(OBJDIR)/%.o,$(DEPDIR)/%.d, $(RTOBJS)))
-
```

Comment:
Funny I first noticed that it was more than comments when @cdsteinkuehler  commented.

Yes even though it would be very nice to have been able to allready have cleaned the work table by now. I will just have to be more patient. And learn how to re base a commit properly.

@mhaberler 
Hopefully Sunday afternoon ....